cmd/alternate_state.go

```diff
@@ -0,0 +1,76 @@
+package cmd
+
+import (
+	"github.com/qlik-oss/corectl/internal"
+	"github.com/qlik-oss/corectl/printer"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+var listAlternateStateCmd = &cobra.Command{
```

Comment:
Since this returns a list of all alternate state it should probably be named `listAlternateStatesCmd`

Reply:
Fixed